community/standards/index.md

```diff
@@ -8,7 +8,7 @@ The Julia community is committed to maintaining a welcoming, civil and construct
 **Be respectful and inclusive.**
 Please do not use overtly sexual language or imagery, and do not attack anyone based on any aspect of personal identity, including gender, sexuality, religion, ethnicity, race, age or ability. Keep in mind that what you write in public forums is read by many people who don't know you personally, so please refrain from making prejudiced or sexual jokes and comments – even ones that you might consider acceptable in private. Ask yourself if a comment or statement might make someone feel unwelcomed or like an outsider.
 
-In particular, do not sexualize the term "Julia" or any other aspects of the project. While "Julia" is a female name in many parts of the world, the programming language is not a person and does not have a gender.
+In particular, do not sexualize the term "Julia" or any other aspects of the project. While "Julia" is a female name in many parts of the world, the programming language is not a person, and should not be spoken about as if it were a person.  In languages where there are gender neutral forms, those should always be used (i.e. it/its instead of she/her).  Even in languages where that is not possible, care can be taken to make clear that one is referring to the language, and not a person. For example, in Spanish, one can speak of "El idioma julia", i.e. "the julia language", which uses the masculine pronouns, instead of just "Julia".
```

Comment:
"El idioma julia" and "the julia language" should have "julia" capitalised to match up with the other uses in the paragraph I think. (Since it's not referring to the executable `julia`.)